src/layers/core/arc-layer/arc-layer.js

```diff
@@ -104,7 +108,8 @@ export default class ArcLayer extends Layer {
     }));
   }
 
-  _getModel(gl) {
+  _getModel() {
```

Comment:
I was thinking we could start supplying `gl` as an argument to `initializeState`, `updateState` etc so that the app doesn't have to keep looking at the context. 

```
intializeState({gl}) {
 ...
```

The API would look simpler and less intimidating if we didn't mention context in most layers I think.

The reason I mention it is that this particular change goes the other way, making context more prominent.